src/Raven.Server/Smuggler/Documents/CsvStreamSource.cs

```diff
@@ -153,13 +155,26 @@ public DatabaseRecord GetDatabaseRecord()
 
         public IEnumerable<DocumentItem> GetDocuments(List<string> collectionsToExport, INewDocumentActions actions)
         {
+            var line = 0;
             while (_csvReader.Read())
             {
+                line++;
+
                 if (ProcessFieldsIfNeeded())
                     continue;
 
                 var context = actions.GetContextForNewDocument();
-                yield return ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                DocumentItem item = null;
+                try
+                {
+                    item = ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                }
+                catch (Exception e)
```

Comment:
This will throw index out of range error, which likely cause the user to think the fault is with us and not the source data.
We need to handle this explicitly.

Reply:
"Fail to parse CSV line {line}, Error:{e.Message}"
Can't make users smart, it's behind my capabilities

Reply:
"The CSV provided to import does not conform to standard rules. Check line {line}. Parse Error: {e.Message}"
That would do it.